docs/docs/monitoring.md

```diff
@@ -86,6 +86,50 @@ When Prometheus is enabled in the status plugin (see [Configuration](./configura
 | last_success_bundle_request    | gauge       | Last successful bundle request in UNIX nanoseconds.    | STABLE |
 | bundle_loading_duration_ns     | histogram   | A histogram of duration for bundle loading.            | STABLE |
 
+## Metrics Overview
+
+OPA provides two ways to access performance metrics:
+
+1. **System-wide metrics** via the `/metrics` Prometheus endpoint - Aggregated metrics across all OPA operations
+2. **Per-query metrics** via API responses with `?metrics=true` - Metrics for individual query executions
+
+These serve different purposes: system metrics for monitoring and alerting, per-query metrics for debugging and optimization.
+
+
+## Accessing Metrics
+
+### System-Wide Metrics (Prometheus Endpoint)
+
+Access aggregated metrics across all OPA operations:
+
+- **URL**: `http://localhost:8181/metrics` (default configuration)
+- **Method**: HTTP GET
+- **Format**: Prometheus text format
+- **Contents**: All counters, timers, histograms, Go runtime metrics
+- **Use case**: Monitoring dashboards, alerting, performance trends
+
+### Per-Query Metrics
+
+Get metrics for individual policy evaluations:
+
+- **Method**: Add `?metrics=true` parameter to API requests
+- **Supported APIs**: `/v1/data`, `/v0/data`, `/v1/query`, `/v1/compile`
+- **Contents**: Query-specific parse, compile, and eval timers
+- **Use case**: Debugging slow queries, performance optimization
+
+Example:
+```http
+POST /v1/data/example?metrics=true HTTP/1.1
+```
+
+For details on interpreting per-query metrics, see [REST API Performance Metrics](./rest-api#performance-metrics).
+
+### Other Metric Sources
```

Comment:
This is more some 'further reading' links to other pages in the docs. Rather than 'other metric sources'.

Reply:
I think we can have a rest-api link and policy performance here for per-query metrics too